data/presets/presets/place/plot.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icon": "triangle-stroked",
+    "fields": [
+        "name"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "place": "plot"
+    },
+    "terms": [
+        "plot"
```

Comment:
You don't actually need to include the word "plot" as a search term - it will be included automatically because it is the tag value.

Instead, add synonyms here like `["tract", "parcel", "lot"]`